.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,26 @@
-## Context
+## Links
 
-<!-- What is the purpose of this PR? -->
-
-[Jira ticket](url)\
+[Jira ticket](url)
 [Design document](url)
 
-<!-- Is there anything left to do before this PR can be merged? -->
-
-<details>
-<summary>To do</summary>
-
-- [ ]
+## Nature of changes
 
-</details>
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update
+- [ ] Deployment update
```

Comment:
Looks good -  do you think there's value in aligning it with the conventional commits spec?

<img width="917" alt="image" src="https://github.com/user-attachments/assets/d0c6a9d3-a523-468e-93e4-0a0f5bdfbfdb" />

e.g.
```suggestion
- [ ] `feat:` - New feature (non-breaking change which adds functionality)
- [ ] `fix:` - Bug fix (non-breaking change which fixes an issue)
- [ ] `docs:` - Documentation update
- [ ] `ci:` - Deployment update
- [ ] `!` - Breaking change (any of the above types, but a change that would cause existing functionality to not work as expected)

Based on [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#examples)
```